packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts

```diff
@@ -82,6 +81,7 @@ export function generateTasks(
    *   - Start services
    *   - Start frontend
    *   - Start backend
+   *   - Install Azure Functions binding extensions
```

Comment:
Does `watch` need to depend on `install extensions`? Or, will `install extensions` CRUDs watched files?

Reply:
No and no. The two tasks can run in parallel.